frontend/src/pages/chat/chat.tsx

```diff
@@ -1,7 +1,38 @@
+import { BackwardButton } from '#libs/components/components.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback, useSearchParams } from '#libs/hooks/hooks.js';
+
 import { ChatSidebar } from './components/components.js';
+import styles from './styles.module.scss';
 
 const Chats: React.FC = () => {
-  return <ChatSidebar />;
+  const [isSidebarShown, setIsSidebarShown] = useSearchParams();
+
+  const handleButtonBackward = useCallback(() => {
+    setIsSidebarShown({ sidebarMode: 'show' });
+  }, [setIsSidebarShown]);
+
+  const isSidebarShownParameter =
```

Comment:
this logic repeats too much, let's create a custom hook for it (for retrieving this search params)

Reply:
done